blog/june-2019-workshop.html

```diff
@@ -0,0 +1,275 @@
+      (<a href="https://www.dropbox.com/sh/ipxmm6twu4p2qo1/AABUdQ0i0UOt46q11Ldgy6z7a/Day%202?dl=0&preview=02_Slicing.pdf&subfolder_nav_tracking=1">slides</a> | <a href="https://github.com/HazyResearch/snorkel-superglue/blob/master/tutorials/WiC_slicing_tutorial.ipynb">tutorial</a>)</li>
+  
+    <li>Introduction to Transformation Functions
+      (<a href="https://www.dropbox.com/sh/ipxmm6twu4p2qo1/AABUdQ0i0UOt46q11Ldgy6z7a/Day%202?dl=0&preview=03_Augmentation.pdf&subfolder_nav_tracking=1">slides</a> | <a href="https://github.com/HazyResearch/snorkel-superglue/blob/master/tutorials/WiC_augmentation_tutorial.ipynb">tutorial</a>)</li>
+  
+    <li>Multi-task and Transfer Learning
+      (<a href="https://www.dropbox.com/sh/ipxmm6twu4p2qo1/AABUdQ0i0UOt46q11Ldgy6z7a/Day%202?dl=0&preview=04_MTL.pdf&subfolder_nav_tracking=1">slides</a> | <a href="">tutorial</a>)</li>
+  
+    <li>Beating GLUE and SuperGLUE with Training Data
+      (<a href="https://www.dropbox.com/sh/ipxmm6twu4p2qo1/AABUdQ0i0UOt46q11Ldgy6z7a/Day%202?dl=0&preview=05_SuperGLUE.pdf&subfolder_nav_tracking=1">slides</a> | <a href="https://github.com/HazyResearch/snorkel-superglue/blob/master/tutorials/COPA_pretrain_tutorial.ipynb">tutorial 1</a> | <a href="https://github.com/HazyResearch/snorkel-superglue/blob/master/tutorials/RTE_slicing_tutorial.ipynb">tutorial 2</a>)</li>
+  
+  </ul>
+
+  Many of the tutorials as posted here were under active development.
+  The workshop attendees asked for updated versions with the initial Snorkel v0.9
+  release along with full GLUE and SuperGLUE code, which we’ll be providing in the new tutorials repo.
+
+
+
+  <h2>Looking Ahead</h2>
+  We’ve been working hard on Snorkel v0.9, and we hope this blog post has convinced you to try it out later this summer.
+  And we want to give our workshop participants a massive thank you for their enthusiasm and feedback.
+  If you’re interested in attending a future workshop or staying up-to-date on the latest developments,
+  we encourage you to
+  <a href="https://groups.google.com/forum/#!forum/snorkel-ml">join our mailing list</a>
+  (infrequent posts, we promise).
+  We’re excited to share Snorkel v0.9 with you and even more excited to hear your feedback,
+  so stay tuned for our full release announcement!
+
+  <br><br>
+
+    <div id="disqus_thread"></div>
+    <script>
+
+      /**
+      *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
+      *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables*/
+      /*
+      var disqus_config = function () {
+      this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
+      this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
+      };
+      */
+      (function () { // DON'T EDIT BELOW THIS LINE
```

Comment:
is Disqus without ads?

Reply:
truly no idea

Reply:
looks like no from http://htmlpreview.github.io/?https://github.com/HazyResearch/snorkel/blob/workshop-blog/blog/june-2019-workshop.html